### PR TITLE
Os bindings

### PR DESCRIPTION
**Pull request by nilnor**
This adds support for OS specific bindings, by allowing for an optional `for_os` sub table in a key map. Included is a crude, incomplete and most likely incorrect set of base bindings for OS X.

### CODE REVIEW

**Comment by nilnor** [comment]
So, no comments on this? I'll merge it soon then. FWIW, someone who actually has an OS X machine around would have to try this and check things out, and ideally work with fleshing out and adapt things. That's not going to be me though for the record.

**Comment by shalabhc** [comment]
Please go ahead. I tried this out on OS X recently and the bindings work as intended. I'll take a shot at adapting them once it's in.

File: lib/howl/bindings.moon [hunk 1]
```diff
@@ -211,12 +219,17 @@ export keystrokes_for = (handler, source = nil) ->
 
   keystrokes
 
-export action_for = (translation, source='editor') ->
+export action_for = (tr, source='editor') ->
+  os = sys.info.os
+  empty = {}
```
Comment (kirbyfan64) [inline lib/howl/bindings.moon]: This use of `empty` feels odd to me. Can't the dict just be left as before like `or {}`?

Comment (nilnor) [inline lib/howl/bindings.moon]: Well, it adds some descriptive value I find, but also I cringed a little bit at creating so many empty tables right after one another, when they needn't be different.

Comment (kirbyfan64) [inline lib/howl/bindings.moon]: @nilnor That makes sense.

**Pull request merged**